data/src/main/java/org/akvo/flow/data/net/FlowApiService.java

```diff
@@ -20,6 +20,16 @@
 
 package org.akvo.flow.data.net;
 
-public interface FlowApiService {
+import org.akvo.flow.data.entity.ApiLocaleResult;
 
+import retrofit2.http.GET;
+import retrofit2.http.Headers;
+import retrofit2.http.Url;
+import rx.Observable;
+
+interface FlowApiService {
```

Comment:
Given that this is focused on syncing data points, would it be more appropriate to call it the `DataSyncService` or something along those lines?  Or do you see this as the definition point for all the API requests going to the backend?